baldrick/plugins/tests/test_github_towncrier_changelog.py

```diff
@@ -14,8 +15,8 @@
     filename = "NEWS.rst"
 
 [ tool.testbot ]
-[ tool.testbot.towncrier_changelog ]
-enabled=true
+  [[ tool.testbot.towncrier_changelog ]]
```

Comment:
Didn't make any difference but I thought this is more consistent with what is in `astropy`.